flutter_app/lib/widgets/return_to_launch_widget.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:imacs/modules/return_to_launch.dart';
+
+class ReturnToLaunchTemplate extends StatefulWidget {
+  final ReturnToLaunch returnToLaunchCommand;
+  final int systemID;
+  final int componentID;
+
+// Needs the ReturnToLaunch object from the return_to_launch dart
+
+  const ReturnToLaunchTemplate(
+      {Key? key,
+      required this.returnToLaunchCommand,
+      required this.systemID,
+      required this.componentID})
+      : super(key: key);
+
+  @override
+  ReturnToLaunchButton createState() => ReturnToLaunchButton();
+}
+
+class ReturnToLaunchButton extends State<ReturnToLaunchTemplate> {
+  String buttonLabel = "Return to Launch";
+
+  void updateButton() {
+    setState(() {
```

Comment:
Instead of updating the text of the button, I think it will be better to have a temporary SnackBar appear